content/docs/user-guide/troubleshooting.md

```diff
@@ -7,6 +7,14 @@ This file uses a special engine feature for the following headers, so that a
 custom anchor link is used. Just add {#custom-anchor} after each title:
 -->
 
+## Failed to pull data from the cloud {#missing-files}
```

Comment:
Will you be adding a link to error.dvc.org/missing-files in iterative/dvc/pull/4339 ?

Reply:
(Please let us know when that PR is approved or merged.)

Reply:
Yes I have that queued right now.